src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,119 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIALIZATION_LENGTH = 10;
+    private T[] array;
+    private int size = 0;
+
+    ArrayList() {
+        array = (T[])new Object[INITIALIZATION_LENGTH];
+    }
+
     @Override
     public void add(T value) {
-
+        if (size == array.length) {
+            makeBiggerArray();
+            array[size] = value;
+            size++;
+            return;
+        }
+        array[size] = value;
+        size++;
     }
 
     @Override
     public void add(T value, int index) {
-
+        size++;
+        if (checkIndex(index)) {
+            throw new ArrayListIndexOutOfBoundsException("Cant add element"
```

Comment:
Replace all hardcoded messages with constants

Reply:
And maybe it would be better to add throw new ArrayListIndexOutOfBoundsException to the checkInIndex method?